build-system/compile/sources.js

```diff
@@ -41,6 +41,8 @@ const COMMON_GLOBS = [
   'third_party/webcomponentsjs/ShadowCSS.js',
   'node_modules/dompurify/dist/purify.es.js',
   'node_modules/promise-pjs/promise.mjs',
+  'node_modules/rrule/dist/esm/src/rrulestr.js',
```

Comment:
With https://github.com/ampproject/amphtml/pull/26123 merged, please add the `package.json` to the list.